grid_unit.pas

```diff
@@ -1420,7 +1420,7 @@ procedure draw_screengrid(canv: TCanvas);      // draw a screen grid if it's wan
 procedure draw_reminders(canv: TCanvas);      // 216a
 
 var
-  memo: TMemo;
+  reminder_memo: TMemo;
```

Comment:
I had to do this rename to avoid a name conflict due to the use of a "with" statement...